change/@ni-spright-components-4795f510-5b35-4ec4-b04d-6935fca41947.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Will you update this usage or do you want me to?

https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/nigel/src/components/sl-nigel/sl-nigel.component.ts&version=GBmaster&line=31&lineEnd=32&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents